### PR TITLE
adding petsird python package

### PR DESCRIPTION
**Pull request by gschramm**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by gschramm** [comment]
@casperdcl @KrisThielemans could add a comment mentioning that you are ok to be a maintainer

**Comment by casperdcl** [comment]
I don't agree with the `-python` suffix, regardless of how many other lang(s) may be distrubuted in future.

**Comment by KrisThielemans** [comment]
Happy to be maintainer.

**Comment by KrisThielemans** [comment]
> I don't agree with the `-python` name suffix, regardless of how many other lang(s) may be distrubuted in future.

I'm not sure what the conda-forge conventions are. I suppose there's not much else than python and C/C++ on conda-forge though. Could use `petsird` for python and `libpetsird` for C++ (as we'll only package the interface I think, if not, not sure what to do for the applications)

**Comment by gschramm** [comment]
@casperdcl changed the name to petsird

**Comment by casperdcl** [comment]
Thanks.

I'm fine with being on the list of recipe maintainers.

**Review by casperdcl — APPROVED** [review]

File: recipes/petsird-python/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  imports:
+    - petsird
+  commands:
+    - pip check
+  requires:
+    - python {{ python_min }}
+    - pip
+
+about:
+  summary: Library and tools for working with PET Emission Tomography Standardization Initiative Raw Data (PETSIRD) data
```
Comment (casperdcl) [inline recipes/petsird-python/meta.yaml:37]:
```suggestion
  summary: Library and tools for working with Positron Emission Tomography Standardization Initiative Raw Data (PETSIRD) data
```

**Review by casperdcl — CHANGES_REQUESTED** [review]

File: recipes/petsird-python/meta.yaml (hunk 1 above)
Comment (gschramm) [reply recipes/petsird-python/meta.yaml:37]: Happy to change this, but this is auto generated from whatever there is on pypi ;)

**Comment by gschramm** [comment]
@conda-forge/help-python ready for review

File: recipes/petsird-python/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{% set name = "petsird" %}
```
Comment (ocefpaf) [inline recipes/petsird-python/meta.yaml:1]:
```suggestion
{% set python_min = python_min|default("3.8") %}
{% set name = "petsird" %}
```

Comment (gschramm) [reply recipes/petsird-python/meta.yaml:1]: @ocefpaf the pypi package says `python >= 3.9`. Can we also have `{% set python_min = python_min|default("3.9") %}` ?

Comment (ocefpaf) [reply recipes/petsird-python/meta.yaml:1]: Yep. You should adjust that to what is in the package metadata.

**Comment by gschramm** [comment]
@ocefpaf I think we are ready to merge

**Pull request merged**